web.go

```diff
@@ -30,5 +30,15 @@ func poweredBy(res http.ResponseWriter, req *http.Request) {
 	}
 	// Print the string to the ResponseWriter
 	hostname, _ := os.Hostname()
-	fmt.Fprintf(res, "Powered by %v\nRelease %v on %v\n", powered, release, hostname)
+	fmt.Fprintf(w, "Powered by %v\nRelease %v on %v\n", powered, release, hostname)
+}
+
+// Log the HTTP request
+func logHandler(w http.ResponseWriter, r *http.Request) {
+	log.Printf("%s %s %s", r.RemoteAddr, r.Method, r.URL)
+}
+
+func rootHandler(w http.ResponseWriter, r *http.Request) {
+	logHandler(w, r)
```

Comment:
Good point: as much as possible, we should have the example-* apps show best practices, and that is better code. I'll add a commit to use that middleware pattern.